Worktrolly/worktrolly
# Dev/orgAPI

### Functionality:
Optimizing APIs to work with Organizations

### Solution:
Added "Sprints" and "Tasks" collection in db from the createNewTaskAPI and startNewSprintAPI.

### Risk level:

### How to test:
1. ng serve
2. Create an organization and team
3. Create sprint and tasks by passing the organization "appKey" from ui
4. Notice changes in database
